### DISCUSSION TITLE
数据集选择

### DISCUSSION THREAD

**Discussion by 8613425400769** [DapengFeng/S3E#23] [Q&A]
在DCL-SLAM当中，请问如果我使用lio_sam的形式的run.launch，<arg name="SYSU_bag" value="$(env HOME)/rosbag-data/S3E/SYSU_LIBRARY.bag"/>和<arg name="bag_file3" value="$(env HOME)/2022_07_22_19_48_01_rect.bag"/>
	<arg name="bag_file4" value="$(env HOME)/2022_07_22_19_54_37_rect.bag"/>
	<arg name="bag_file5" value="$(env HOME)/2022_07_22_20_01_00_rect.bag"/>
  	<node name="player3" pkg="rosbag" type="play" output="screen" args="-r 1 -p a -q -d 2 $(arg bag_file3)"/>
	<node name="player4" pkg="rosbag" type="play" output="screen" args="-r 1 -p b -q -d 2 $(arg bag_file4)"/>是哪S3E中哪几个数据集？

**Comment by DapengFeng** [comment]
请问您具体指哪个bag? 新的S3E已经迁移到ROS2了，可以用https://ternaris.gitlab.io/rosbags/topics/convert.html 转成ROS1

**Comment by 8613425400769** [comment]
bag问题已经解决了，但是现在就是用fast-lio2做前端的时候，机器人内部自己的闭环没有优化，这是正常的吗？而且又时候会出现process died，这个只是偶尔会，主要还是intra的闭环没有优化，图片是效果图，可以解释一下是为什么吗，辛苦你了，这个slam系统确实比较有意思






......
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "DapengFeng/S3E"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月18日(星期五) 晚上10:22
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [DapengFeng/S3E] 数据集选择 (Discussion #23)





 
请问您具体指哪个bag?

**Reply by DapengFeng** [reply]
可能传输出问题了，图像看不到

**Comment by 8613425400769** [comment]
传输出问题是指哪里？有解决的方法吗






......
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "DapengFeng/S3E"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月18日(星期五) 晚上11:07
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [DapengFeng/S3E] 数据集选择 (Discussion #23)





 
可能传输出问题了，图像看不到

**Reply by DapengFeng** [reply]
看不到您说的效果图

![屏幕截图 2024-10-18 234600](https://github.com/user-attachments/assets/ba1b1180-7b3c-44cd-800e-2e2f1345a890)

**Comment by 8613425400769** [comment]
附件是照片，可以看到吗




......
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "DapengFeng/S3E"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月18日(星期五) 晚上11:48
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [DapengFeng/S3E] 数据集选择 (Discussion #23)





 
看不到您说的效果图
 
2024-10-18.234600.png (view on web)

**Comment by 8613425400769** [comment]
照片放在附件里面，能看到吗？或者能不能加个联系方式我发给你看看&nbsp;&nbsp; LXN2544376383，辛苦您了



......
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "DapengFeng/S3E"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月18日(星期五) 晚上11:48
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [DapengFeng/S3E] 数据集选择 (Discussion #23)





 
看不到您说的效果图
 
2024-10-18.234600.png (view on web)

**Comment by 8613425400769** [comment]
大佬，能出一张关于DCL-SLAM用fast-lio2前端的代码逻辑图吗，对于代码的理解有点不太懂，可以吗？或者能否加一下联系方式沟通一下，辛苦您了





......
***@***.***



&nbsp;

**Reply by DapengFeng** [reply]
由于我没有实质性参与DCL-SLAM的框架设计，具体细节我了解得不多，你可以参考一下论文：https://arxiv.org/pdf/2210.11978

**Comment by 8613425400769** [comment]
附件有一个rviz的显示图，图中并没有执行机器人内部闭环的优化，请问这方面能给我提供帮助吗




......
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "DapengFeng/S3E"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月21日(星期一) 上午10:25
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [DapengFeng/S3E] 数据集选择 (Discussion #23)





 
由于我没有实质性参与DCL-SLAM的框架设计，具体细节我了解得不多，你可以参考一下论文：https://arxiv.org/pdf/2210.11978

**Reply by DapengFeng** [reply]
您说的是这个问题 https://github.com/PengYu-Team/DCL-SLAM/issues/35 吗？

**Reply by DapengFeng** [reply]
![image](https://github.com/user-attachments/assets/7f6050e3-5a5b-47c1-91a3-51764ed2536d)

**Comment by 8613425400769** [comment]
......
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "DapengFeng/S3E"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月21日(星期一) 下午4:41
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [DapengFeng/S3E] 数据集选择 (Discussion #23)





 
image.png (view on web)

**Comment by 8613425400769** [comment]
请问，能找出是什么问题吗，即使我什么都不改跑这个数据集一样没法达到视频中的那种效果



......
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "DapengFeng/S3E"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月21日(星期一) 下午4:41
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [DapengFeng/S3E] 数据集选择 (Discussion #23)





 
image.png (view on web)
